### ISSUE TITLE
Bug/Incompatibility with Create

### ISSUE THREAD

**Issue by aptom203** [spoorn/sodium-forge#166]
### Expected Behavior
Create machines rendered with moving parts.

### Actual Behavior
Structures attached to bearings, gantries, pistons, pulleys etc from Create mod fail to render while moving.

### Reproduction Steps
Attach a structure of any sort to a Create bearing, gantry, pulley or piston. When the Create mechanism is activated, the structure fails to render. The structure re-appears when the mechanism is disabled or destroyed.

### Mods list
Bookshelf (by DarkhaxDev)
Mana and Artifice (by AWildCanadianEh)
Just Enough Items (JEI) (by mezz)
ZeroCore 2 (by ZeroNoRyouki)
Mekanism Additions (by bradyaidanc)
Refined Storage Addons (by raoulvdberge)
RFTools Base (by McJty)
CC: Tweaked (by SquidDev)
Light Overlay (Rift/Forge/Fabric) (by shedaniel)
Iron Jetpacks (by BlakeBr0)
FTB Essentials (Forge) (by FTB)
ConnectedTexturesMod (by tterrag1098)
KubeJS Create (by LatvianModder)
Pam's HarvestCraft 2 - Food Extended (by pamharvestcraft)
Ender Storage 1.8.+ (by covers1624)
Environmental Tech (by ValkyrieofNight)
JEITweaker (by Jaredlll08)
Absent by Design (by Lothrazar)
Simple Generators (by TeamKrypticLink)
ReAuth (by TechnicianLP)
Storage Drawers (by Texelsaur)
MrCrayfish's Furniture Mod (by MrCrayfish)
AppleSkin (by squeek502)
Refined Storage: Requestify (by Buuz135)
Industrial Foregoing (by Buuz135)
Cucumber Library (by BlakeBr0)
Architectury API (Forge) (by shedaniel)
FTB Library (Forge) (by FTB)
Thermal Locomotion (by TeamCoFH)
FTB Teams (Forge) (by FTB)
Reliquary v1.3 (by P3pp3rF1y)
Dungeon Crawl (by xiroc_)
Cloth Config API (Forge) (by shedaniel)
Waystones (by BlayTheNinth)
Thermal Innovation (by TeamCoFH)
ValkyrieLib (by ValkyrieofNight)
EnderTanks (by ShetiPhian)
Rats (by alex_khaan)
Spice of Life: Carrot Edition (by lordcazsius)
Waila Harvestability (by squeek502)
ATO - All the Ores (by ATMTeam)
Wawla - What Are We Looking At (by DarkhaxDev)
U Team Core (by HyCraftHD)
Silent Lib (silentlib) (by SilentChaos512)
RFTools Builder (by McJty)
RFTools Control (by McJty)
Runelic (by DarkhaxDev)
Trash Cans (by SuperMartijn642)
Citadel (by alex_khaan)
TiCBC - Tinkers Better Compat (by ianm1647)
XNet Gases (by Terrails)
SuperMartijn642's Core Lib (by SuperMartijn642)
Titanium (by Buuz135)
RFTools Utility (by McJty)
GeckoLib (by ThanosGecko)
Silent Gear (by SilentChaos512)
Dark Utilities (by DarkhaxDev)
Pam's HarvestCraft 2 - Trees (by pamharvestcraft)
Thermal Cultivation (by TeamCoFH)
Environmental Core (by ValkyrieofNight)
KubeJS Thermal (by LatvianModder)
Immersive Engineering (by BluSunrize)
KubeJS Mekanism (by LatvianModder)
Flux Networks (by sonar_sonic)
Chisel (by tterrag1098)
Useful Backpacks (by HyCraftHD)
Scorge (by illyohs)
Halogen (by spoorn)
Tinkers Construct (by mDiyo)
Macaw's Trapdoors (by sketch_macaw)
Immersive Petroleum (by Flaxbeard)
Crafting Tweaks (by BlayTheNinth)
Create (by simibubi)
Patchouli (by Vazkii)
Mantle (by mDiyo)
Building Gadgets (by Direwolf20)
Mana Gear (by Partonetrain)
Structurize (by H3lay)
JourneyMap (by techbrew)
KubeJS (Forge) (by LatvianModder)
Macaw's Windows (by sketch_macaw)
Morpheus (by Quetzi)
Extreme Reactors (by ZeroNoRyouki)
CraftTweaker (by Jaredlll08)
Borealis (by ILIKEPIEFOO2)
OpenBlocks Elevator (by vsngarcia)
Mekanism (by bradyaidanc)
Refined Storage (by raoulvdberge)
McJtyLib (by McJty)
Mekanism Generators (by bradyaidanc)
Mekanism Tools (by bradyaidanc)
RFTools Storage (by McJty)
InfinityLib (by InfinityRaider)
Farming for Blockheads (by BlayTheNinth)
Botania (by Vazkii)
CodeChicken Lib 1.8.+ (by covers1624)
Solar Flux Reborn (by Zeitheron)
Akashic Tome (by Vazkii)
Polymorph (Forge) (by TheIllusiveC4)
NetherPortalFix (by BlayTheNinth)
Thermal Foundation (by TeamCoFH)
KubeJS Immersive Engineering (by LatvianModder)
Project Red - Core (by MrTJP)
Pam's HarvestCraft 2 - Crops (by pamharvestcraft)
JAOPCA (by TheLMiffy1111)
CoFH Core (by TeamCoFH)
Cooking for Blockheads (by BlayTheNinth)
Thermal Expansion (by TeamCoFH)
RFTools Power (by McJty)
Quark (by Vazkii)
Just Enough Resources (JER) (by way2muchnoise)
Cosmetic Armor Reworked (by LainMI)
Macaw's Fences and Walls (by sketch_macaw)
Botany Pots (by DarkhaxDev)
ExtraStorage (by 3divad99)
Macaw's Bridges (by sketch_macaw)
KubeJS Borealis (by ILIKEPIEFOO2)
Armor Status HUD Renewed (by Bobsans)
Tinkers' Mechworks (by mDiyo)
Better Advancements (by way2muchnoise)
Corail Tombstone (by Corail_31)
Nature's Compass (by Chaosyr)
Silent's Mechanisms (by SilentChaos512)
Quark Oddities (by Vazkii)
Iron Chests (by ProgWML6)
Mob Grinding Utils (by vadis365)
AutoRegLib (by Vazkii)
FTB Chunks (Forge) (by FTB)
Fast Leaf Decay (by olafskiii)
Cyclic (by Lothrazar)
RFTools Dimensions (by McJty)
Rhino (by LatvianModder)
Curios API (Forge) (by TheIllusiveC4)
Project Red - Integration (by MrTJP)
Project Red - Illumination (by MrTJP)
Budschie's Morph Mod (by budschie_)
Pam's HarvestCraft 2 - Food Core (by pamharvestcraft)
Macaw's Roofs (by sketch_macaw)
Immersive Posts (by TwistedGate)
XNet (by McJty)
Extra Disks (by MelanX)
Flywheel (by jozufozu)
Mouse Tweaks (by YaLTeR)
CB Multipart (by covers1624)
Macaw's Doors (by sketch_macaw)
Controlling (by Jaredlll08)
Macaw's Furniture (by sketch_macaw)
Hwyla (by TehNut)
ShetiPhianCore (by ShetiPhian)
FTB Ultimine (Forge) (by FTB)
JEI Integration (by SnowShock35)
Iron Furnaces [FORGE] (by XenoMustache)
BobLib (by Bobsans)

### System Information
You can obtain this information from the in-game debug screen or through using [DxDiag](https://support.microsoft.com/en-us/help/4028644/windows-open-and-run-dxdiagexe) on Windows.

- Java Version: jre-8u301-windows-x64
- CPU: AMD Ryzen 3 2200G with Radeon Vega Graphics, 3500 Mhz, 4 Core(s), 4 Logical Processor(s)
- GPU: Radeon RX Vega 56
- Sodium Reforged Version: sodium-forge-2.9.10+rev.ea6353c.jar

**Comment by Gogo08190** [comment]
I have the same problem !

![2021-09-01_20 39 35](https://user-images.githubusercontent.com/41335079/131725801-04fac18e-2ab5-4975-a641-ef67204cb840.png)

**Comment by FlameyFaucet** [comment]
if you want to keep using create i suggest downgrading to create 0.3.2c and using a flywheel ver that isn't 0.2.3

**Comment by Gogo08190** [comment]
Thanks !

**Comment by MrReps** [comment]
it's a known bug suposly fix in the last rev, but aparently no

**Comment by Yarden-zamir** [comment]
Is it on their end? Do they know it's not fixed?

**Comment by MrReps** [comment]
don't know no news of the owner from since this time

**Comment by Kqmu** [comment]
Still a problem in Versions:
**create-mc1.16.5_v0.3.2d.jar**
**flywheel-1.16-0.2.3.jar**
**sodium-forge-2.9.10+rev.ea6353c.jar**

![2021-09-05_22 10 10](https://user-images.githubusercontent.com/70139586/132150843-99719c4b-990e-4ac5-ae78-827fce91b38a.png)

**Comment by FlameyFaucet** [comment]
yes, it is, in fact, still a bug. i believe the creator of flywheel acknowledged this bug as a side effect of a change they made that fixed optifine compatibility, and they are working to make out what happened and fixing it.

see my comment about 5 posts above if you want to keep playing with create w/sodium
